Examples/Files/nps-icons.xcassets/nps-trailhead.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "16px-nps-trailhead.png",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "nps-trailhead.png",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "64px-nps-trailhead.png",
```

Comment:
If the `@1x` asset is 16px, the `@3x` asset should be 48px (`16 × 3`), not 64px.

Reply:
@jmkiley is this resolved?